pkg/subctl/cmd/gather/cni.go

```diff
@@ -82,6 +82,8 @@ var networkPluginCNIType = map[string]string{
 	"unknown":       typeUnknown,
 }
 
+var cniResource resourceInfo
```

Comment:
I don’t think this needs to be a global variable, since it’s only used in `CNIResources()`.